less/index.less

```diff
@@ -48,129 +50,210 @@ table {
 	border-spacing: 0;
 }
 
-/* Set every element's box-sizing to border-box */
+// Set every element's box-sizing to border-box
 * {
     box-sizing: border-box;
 }
 
 html, body {
     height: 100%;
     font-family: 'Titillium Web', sans-serif;
+    font-size: 62.5%;
 }
 
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
+    color: @header-color;
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
 
 .container {
     width: 880px;
     margin: auto;
```

Comment:
Changing the width to a percentage allows it to dynamically resize based on the screen size.